warehouse/upload.go

```diff
@@ -445,6 +442,40 @@ func (job *UploadJobT) resolveIdentities(populateHistoricIdentities bool) (err e
 	return idr.Resolve()
 }
 
+func (job *UploadJobT) updateTableSchema(tName string, tableSchemaDiff warehouseutils.TableSchemaDiffT) (err error) {
+	pkgLogger.Infof(`[WH]: Starting schema update for table %s in namespace %s of destination %s:%s`, tName, job.warehouse.Namespace, job.warehouse.Type, job.warehouse.Destination.ID)
+
+	if tableSchemaDiff.TableToBeCreated {
+		err = job.whManager.CreateTable(tName, tableSchemaDiff.ColumnMap)
+		if err != nil {
+			pkgLogger.Errorf("Error creating table %s on namespace: %s, error: %v", tName, job.warehouse.Namespace, err)
+		}
+		return err
```

Comment:
Write a comment here saying - Since we are creating a table, we don't need to run alter columns.